audb/core/publish.py

```diff
@@ -171,60 +171,81 @@ def _find_media(
     num_workers: int,
     verbose: bool,
 ) -> set[str]:
-    r"""Find archives with new, altered or removed media and update 'deps'."""
+    """Find archives with new, altered or removed media and update 'deps'.
+
+    The function alters the dependency table entries of ``deps`` in place
+    by adding new entries for new media files,
+    removing entries for removed media files,
+    and updating entries for altered media files.
+    It further collects all media archives,
+    that need to be uploaded
+    after they are created or changed.
+
+    Args:
+        db: database
+        db_root: path to root of database
+        db_root_files: (media) files in root of database
```

Comment:
I find the name `db_root_files` a little confusing. When digging into `_get_root_files` the files are actually not files in the root directory, of `db_root`, but rather are all files that make up the database as the directory listing is recursive. When talking about `root_files`, my intuition would be that one talks about metdata files that directly sit in `db_root` omitting subdirectories (`db.yaml`, `db.parquet` and some table files) . My understanding is that the meaning here is rather "all files that are in db_root".

Reply:
I changed to
```
db_root_files: all files in root of database
```